client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -141,8 +141,12 @@ const CheckoutThankYou = React.createClass( {
 			const site = this.props.selectedSite.slug;
 
 			if ( purchases.some( isPlan ) ) {
-				page( `/plans/${ site }` );
-			} else if ( purchases.some( isDomainProduct ) || purchases.some( isDomainRedemption || purchases.some( isSiteRedirect ) ) ) {
+				page( `/plans/my-plan/${ site }` );
+			} else if (
+				purchases.some( isDomainProduct ) ||
+				purchases.some( isDomainRedemption ||
+				purchases.some( isSiteRedirect ) )
```

Comment:
On line 147, you are missing closing `)` right before `||` and on line 148 you have additional `)`. :)